website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -26,6 +26,12 @@ Use inline alerts when the message is contextual or specific to the section it
 
 Use compact alerts when a less prominent alert is needed.
 
+### Ordering Alerts by type
+
+Alert types have an implied hierarchy and should be ordered starting from `page` and descending in hierarchy to `compact`.
+
+![Ordering alerts of different types](/assets/components/alert/ordering-alerts-type.png)
```

Comment:
not to be picky, but maybe this image could be something like this to convey even more the hierarchy?

<img width="1072" alt="screenshot_3499" src="https://github.com/hashicorp/design-system/assets/686239/c472da1e-e48d-4ba9-81d4-77bef8362996">

Reply:
Good suggestion, I've updated this image to add a bit more implied hierarchy.